### PR TITLE
Avoid conflict in custom channel

### PR DESCRIPTION
**Pull request by keichwa**
https://github.com/SUSE/spacewalk/issues/16652

# Description

Short summary of why you created this PR

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

**Comment by keichwa** [comment]
@mcalmer do you think it would make sense to add this information to our documentation?  If yes, I'd adjust it for the Troubleshooting section of our Admin Guide.  Or should we add it to the so-called Cookbook?

Or would it better to offer this information somehow externally?

**Comment by mcalmer** [comment]
I think it makes sense to have it in the trouble shoot section.
It happens always if we must set the option "--with-custom-channels" for mgr-create-bootstrap-repo script.
This happens internally for native Red Hat and RES as we cannot setup full repositories as vendor channels with defined IDs.
Maybe also for Ubuntu 18, would need to check it.
For native Red Hat we must use custom repos and Expanded Support does not provide all packages and you have to import packages into a custom channel.

I am not sure if it will not go away 100% with the bundle, as I think we still allow to not use the bundle. But maybe we do not provide normal salt packages for RedHat anymore.

File: modules/administration/pages/tshoot-custom-channel.adoc [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+Result: Clean and functional boostrap repo will be created
+////
+
+When setting up a custom channel with conflicting packages features such as creating a bootstrap repository can result in undefined behavior and make registering clients fail.
+
+For example, conflicting packages with higher version numbers could be
+included into the bootstrap repository.
+Such packages (for example, [package]``python3-zmq`` or [package]``zeromq``) may corrupt the creation of the bootstrap repository or cause issues during bootstrap of the client.
+
+When the custom channel (for example, an EPEL channel) is added below the parent channel, issues with conflicting packages can not be solved directly.
```
Comment (mcalmer) [inline modules/administration/pages/tshoot-custom-channel.adoc:35]:
```suggestion
When the custom channel (for example, an EPEL channel) is added below the parent vendor channel, issues with conflicting packages can not be solved directly.
```

File: modules/administration/pages/tshoot-custom-channel.adoc [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+The way how to solve this is to separate the custom channel from the parent channel.
```
Comment (mcalmer) [inline modules/administration/pages/tshoot-custom-channel.adoc:36]:
```suggestion
The way how to solve this is to separate the custom channel from the vendor channel.
```

**Review by mcalmer — APPROVED** [review]
Looks good so far. I just added 2 small suggestions to make the problem more clear

**Comment by keichwa** [comment]
> Looks good so far. I just added 2 small suggestions to make the problem more clear

thank you!

File: modules/administration/pages/tshoot-custom-channel.adoc [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+Result: Clean and functional boostrap repo will be created
+////
+
+When setting up a custom channel with conflicting packages features such as creating a bootstrap repository can result in undefined behavior and make registering clients fail.
+
+For example, conflicting packages with higher version numbers could be
+included into the bootstrap repository.
+Such packages (for example, [package]``python3-zmq`` or [package]``zeromq``) may corrupt the creation of the bootstrap repository or cause issues during bootstrap of the client.
+
+When the custom channel (for example, an EPEL channel) is added below the parent vendor channel, issues with conflicting packages can not be solved directly.
```
Comment (keichwa) [inline modules/administration/pages/tshoot-custom-channel.adoc:35]:
```suggestion
When the custom channel (for example, an EPEL channel) is added below the parent vendor channel, issues with conflicting packages cannot be solved directly.
```

**Review by jcayouette — APPROVED** [review]

**Review by 0rnela — APPROVED** [review]

File: .changelog [hunk 4]
```diff
@@ -4,6 +4,11 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)#Changelog_section_.28.25changelog.29
 
+- Add troubleshooting section about avoiding package conflicts with custom
+  channels
+- merge translations of installation and upgrade guide into the new
+  combinded book
+- removed end of life caasp references from documentation
```
Comment (keichwa) [inline .changelog:11]:
```suggestion
```

**Pull request merged**